### PR TITLE
Prettier: Just prettier@2 💅

### PR DESCRIPTION
**Pull request by sirreal**
Discussion: p4TIVU-9oH-p2

It's _just_ prettier.

Forget forks.

This PR also adopts the Prettier 2 default of `arrowParens: always`, i.e. parens will wrap arguments in an arrow function like this: `(singleArg) => result`.

Read more about prettier 2.0 in the release notes:

https://prettier.io/blog/2020/03/21/2.0.0.html

Closes #41114

* Upgrade to `prettier@2.0.4`
* `yarn reformat-files`

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -2,9 +2,3 @@ useTabs: true
 tabWidth: 2
 printWidth: 100
 singleQuote: true
-trailingComma: es5
-bracketSpacing: true
-parenSpacing: true
-jsxBracketSameLine: false
-semi: true
-arrowParens: avoid
```
Comment (sirreal) [inline .prettierrc:10]: These are default values + `parenSpacing` which was what the `wp-prettier` fork added.

**Comment by noahtallen** [comment]
Does prettier now support the WP standard through various options?? (I see some things here are different, like spaces inside parentheses)

**Comment by simison** [comment]
@noahtallen I think for now slack for context: p1587053173207800-slack-team-calypso

**Comment by noahtallen** [comment]
lol

i am not ready for this

**Comment by sirreal** [comment]
This initiative seems to have fizzled out after discussion: p4TIVU-9oH-p2

Superseded by https://github.com/Automattic/wp-calypso/pull/41114

**Pull request closed without merging**